BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -178,6 +178,7 @@ - (instancetype)init
                                                                  APIBaseURL:_APIBaseURL
                                                              APIAuthBaseURL:_APIAuthBaseURL
                                                                queueManager:_queueManager];
+        _OAuth2Session.userAgentPrefix = self.userAgentPrefix;
```

Comment:
Looking back, these will probably be nil since they are in the init. We can remove these to be cleaner.